processing/data_collection/gazette/spiders/pe_jaboatao_dos_guararapes.py

```diff
@@ -0,0 +1,48 @@
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
+
```

Comment:
```suggestion
from datetime import datetime

from dateparser import parse
from scrapy import Request

```

See more in [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports)